googledatastore/conf/googledatastore.properties

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) 2015 YCSB contributors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you
+# may not use this file except in compliance with the License. You
+# may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing
+# permissions and limitations under the License. See accompanying
+# LICENSE file.
+
+#
+# Sample property file for Google Cloud Datastore DB client 
+
+## Mandatory parameters
+# See this link for instructions about setting up Google Cloud Datastore and
+# authentication:
+# https://cloud.google.com/datastore/docs/getstarted/start_java/
+#
+# After you setup your environment, you will have 3 pieces of information ready:
+# datasetId, service account email, and a private key file. These must be
+# configured via the properties below
+#
+# googledatastore.datasetId=<string id of your dataset>
+# googledatastore.privateKeyFile=<full path to your private key file>
+# googledatastore.serviceAccountEmail=<Your service account email>
```

Comment:
Yep, I agree this variable name should be serviceAccountEmail. And thanks for the catch!